components/Images/Images.css

```diff
@@ -0,0 +1,15 @@
+
+.img_responsive{
+     width:50rem;
+     height:30rem;
```

Comment:
Hey Shrinkhla actually the point of the responsive image is that the image will fit inside any container and take space according to the container's width which will make the image responsive. So giving the image fixed width and height might not help. Maybe try `width:100%; height:auto;` .

Reply:
thanks for clearing this..Will improve this.